src/i18n/translations.json

```diff
@@ -141,6 +140,11 @@
         "wrong.password": "The password is invalid",
         "invalid.email": "The email address is badly formatted",
         "weak.password": "Password should be at least 6 characters",
+        "welcome": "Welcome {name}",
+        "mail.send.button.done": "Done",
+        "reset.password.confirmation.title": "Password Reset Email Sent",
+        "reset.password.confirmation.first.text": "An email has been sent to your email address,",
+        "reset.password.confirmation.second.text": "Follow the directions in the email to reset your password."
```

Comment:
you need to be careful and test that the app is working because here you forgot to put the coma at the end and this will lead to failed to compile error